hbase-it/src/main/java/org/apache/hadoop/hbase/chaos/util/ChaosMonkeyRunner.java

```diff
@@ -24,18 +24,23 @@
 import org.apache.commons.lang3.StringUtils;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.hbase.HBaseConfiguration;
+import org.apache.hadoop.hbase.HBaseInterfaceAudience;
 import org.apache.hadoop.hbase.HConstants;
 import org.apache.hadoop.hbase.IntegrationTestingUtility;
 import org.apache.hadoop.hbase.TableName;
 import org.apache.hadoop.hbase.chaos.factories.MonkeyFactory;
 import org.apache.hadoop.hbase.chaos.monkies.ChaosMonkey;
 import org.apache.hadoop.hbase.util.AbstractHBaseTool;
 import org.apache.hadoop.util.ToolRunner;
+import org.apache.yetus.audience.InterfaceAudience;
+import org.apache.yetus.audience.InterfaceStability;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.apache.hbase.thirdparty.com.google.common.collect.Sets;
 import org.apache.hbase.thirdparty.org.apache.commons.cli.CommandLine;
 
+@InterfaceAudience.LimitedPrivate({HBaseInterfaceAudience.CHAOS, HBaseInterfaceAudience.TOOLS})
```

Comment:
Maybe I missed discussion around including `HBaseInterfaceAudience.TOOLS` for `ChaosMonkeyRunner`. Hope this is as per the discussions.

Reply:
@virajjasani did discussion internally as ChaosMonkeyRunner can be used as tool (can be used from command line), still needed confirmation. waiting from others.

Reply:
IMHO it should be fine to add TOOLS to ChaosMonkeyRunner.

CC: @apurtell @busbey @mihir6692